src/components/listItens/CountItens.js

```diff
@@ -0,0 +1,24 @@
+import React,{useState} from "react";
+import { BsPlusSquare } from "react-icons/bs";
+import { BsDashSquare } from "react-icons/bs";
+
+function CountItens({stock="10",initial="1",onAdd}){
+       
+    const [itensNumber, setIntensNumber]=useState(1)
+    
+    return(
+     
+        <div className="countItens"> 
+        <div className="countItensContainer" >
+        {itensNumber<=1?<BsDashSquare/>:<BsDashSquare  className= "dashButton" onClick={()=>{setIntensNumber(anterior=>anterior-1)}} />}
```

Comment:
Oi Alex e Cíntia!

Sugestão para que vocês consigam fazer a funcionalidade de fazer o contador parar quando chegar ao máximo em estoque:

1 - Adicione nessa função do onclick uma validação da quantidade de item e só some se esse valor for menor:

```
<BsDashSquare 
      className= "dashButton"
      onClick={()=>{
           if( itensNumber < stock ) {
              setIntensNumber(anterior=>anterior-1)
            }
      }} 
 />
```

Reply:
Oi Ângela, já fizemos as alterações incluindo a validação do onclick.